pyperformance/data-files/benchmarks/bm_ctypes/cmodule.c

```diff
@@ -0,0 +1,54 @@
+#include <Python.h>
+
+#if defined(_WIN32) || defined(__CYGWIN__)
+#define EXPORTED_SYMBOL __declspec(dllexport)
+#else
+#define EXPORTED_SYMBOL
```

Comment:
Maybe export the symbol on other platforms too? 
The following should work for gcc and clang:
```suggestion
#define EXPORTED_SYMBOL __attribute__((visibility ("default")))
```

Reply:
You can also use the one which Python uses: `Py_EXPORTED_SYMBOL` and avoid this altogether.

Reply:
Unfortunately, `Py_EXPORTED_SYMBOL` doesn't exist in Python 3.8.  And the explicit visibility isn't necessary on gcc or clang where visible is the default.

I could copy the full implementation of `Py_EXPORTED_SYMBOL` in CPython here and use it, though, if that's just "less surprising".

Reply:
> Py_EXPORTED_SYMBOL doesn't exist in Python 3.8. 

Oh, I didn't know that, probably best left as it is then.